src/utils/upload/processUpload.js

```diff
@@ -196,5 +186,41 @@ const processUpload = async (filesList, sampleType, samples, activeProjectUuid,
   });
 };
 
-export { compressAndUploadSingleFile, metadataForBundle, renameFileIfNeeded };
-export default processUpload;
+const bundleToFile = async (bundle, technology) => {
+  // This is the first stage in uploading a file.
+
+  let filename;
+  if (bundle.path) {
+    // trim path to just the file and its folder
+    filename = _.takeRight(bundle.path.split('/'), 2).join('/');
+  } else {
+    filename = bundle.name;
+  }
```

Comment:
suggestion - make this into a ternary operator?

Reply:
Yeah, this is probably neater